easybuild/easyconfigs/n/nanocompore/nanocompore-1.0.0rc3-2-intel-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,65 @@
+easyblock = 'PythonBundle'
+
+name = 'nanocompore'
+version = '1.0.0rc3-2'
+versionsuffix = "-Python-%(pyver)s"
+
+homepage = 'https://nanocompore.rna.rocks/'
+description = "Nanocompore identifies differences in ONT nanopore sequencing raw signal corresponding to RNA modifications by comparing 2 samples"
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+    ('tqdm', '4.47.0'),
+    ('matplotlib', '3.2.1', versionsuffix),
+    ('Seaborn', '0.10.1', versionsuffix),
+    ('scikit-learn', '0.23.1', versionsuffix),
+    ('statsmodels', '0.11.1', versionsuffix),
+    ('poetry', '1.0.9', versionsuffix),
+    ('PyYAML', '5.3'),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+# poetry has unnecessarily restrictive dependencies
+local_nanocompore_preinstallopts = """sed -i'' 's/\^/>=/g' pyproject.toml && """
+local_nanocompore_preinstallopts += """sed -i'' 's/~/>=/g' pyproject.toml && """
+
+
+local_bedparse_preinstallopts = """sed -i'' 's/argparse//' setup.py && """
+exts_list = [
+    ('pyfaidx', '0.5.9.1', {
+        'checksums': ['eda8af04ba4da4fd63fdc35a62e0e41dfc06aa1a511728dfbdd7707e3b382855'],
+    }),
+    ('bedparse', '0.2.3', {
+        'preinstallopts': "sed -i'' 's/argparse//' setup.py && ",
+        'checksums': ['d1597b3de525a8d2e4ffe9ca73b2d6b59c1f54f0fe3ad248b42c2f36e50ce8f3'],
+    }),
+    (name, version, {
+        'preinstallopts': "sed -i'' 's/\^/>=/g' pyproject.toml && sed -i'' 's/~/>=/g' pyproject.toml && ",
```

Comment:
you should use the defined `local_nanocompore_preinstallopts` here.

Reply:
changed